app/controllers/receive_package.js

```diff
@@ -133,11 +134,25 @@ export default Ember.Controller.extend({
       if(this.get("hasErrors")) {
         this.get("package").rollbackAttributes();
       }
-      this.transitionToRoute("review_offer.receive");
+      var _this = this;
+      var loadingView = getOwner(this).lookup('component:loading').append();
+      var pkg = this.get("package");
+      var inventoryNumber = pkg.get('inventoryNumber');
+      pkg.set('inventoryNumber', null);
```

Comment:
I think inventory_number should b removed if the package is in missing state.
This will remove inventory_number every time User clicks cancel button.

What do you think??